spec/cancan/ability_spec.rb

```diff
@@ -431,6 +431,20 @@ class Container < Hash; end
     end
   end
 
+  it "raises a custom exception if ability is unauthorized to perform a certain action and is given a custom error class" do
```

Comment:
I don't like this test very much. I'd like more to have one test for each expectation and to actually check that `is_expected.to raise_error(MyCustomAccessDenied)` https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/raise-error-matcher